Packs/CommonTypes/IndicatorTypes/reputation-email.json

```diff
@@ -26,24 +26,10 @@
   "contextValue": "Address",
   "excludedBrands": [],
   "defaultMapping": {
-    "blocked": {
-      "simple": "",
-      "complex": {
-        "root": "Account",
-        "filters": [],
-        "accessor": "Blocked",
-        "transformers": [
-          {
-            "operator": "uniq",
-            "args": {}
-          }
-        ]
-      }
-    },
     "domainname": {
       "simple": "",
       "complex": {
-        "root": "Account",
```

Comment:
@anara123 looks like an historic mistake where 'Account' is root instead of 'Email'. fixing might BC. Do we want to fix it?